CONTRIBUTING.md

```diff
@@ -6,3 +6,74 @@ repository before making a change.
 
 Please note we have a [Code of Conduct](CODE_OF_CONDUCT.md), please follow it
 in all your interactions with the project.
+
+
+# Contributing to the blog
+
+First of all, thank you for wanting to contribute for our blog &#128077;. This section contains all the information you need to know to contribute to our blog page. Should you have any questions, feel free to contact us and we will try our best in answering them.
+
+
+## Before writing
+### Cloning and creating a new branch
+If you have decided to contribute to our blog, the first thing you need to do is clone the repository and creating a new branch where you will be writing our post. In other words, you should run the following commands:
+```
+git clone https://github.com/coderdojobraga/shuriken.git
+cd shuriken
+git switch -c your-branch-name
+```
+where `your-branch-name` should be your initials (first letter of the your first and last name) folowed by `/blog/your-post-topic` . For example, if your name is João Silva and you are writing about how to make an hello world in python, your branch name would be something like  `js/blog/python-hello-world`.
+
+#### Note for those not a part of the Coderdojo Braga team
+
+If you are not a member of our team on Github, then instead of creating a branch you need to fork this project. If you don't know how to do that, please [refer to this guide](https://docs.github.com/pt/enterprise-cloud@latest/get-started/quickstart/fork-a-repo).
```

Comment:
Here, it is before making a branch, you need to do a fork. The separated branch is still required.